scenario11/resources/user.properties

```diff
@@ -18,28 +18,28 @@
 #serverHost=localhost
 
 
-serverHost=$serverHost
-serverPort=$serverPort
+serverHost=is550.qa.com
+serverPort=443
```

Comment:
Can you please remove the hard coded values in the user properties file.
eg: server host, port, tomcat host, port etc.